changelogs/fragments/378.yml

```
@@ -0,0 +1,7 @@
+---
+trivial:
+  - node_exporter - update build dependencies for node-exporter-extras package
```

```suggestion
minor_changes:
  - node_exporter - minimum version 1.5.0, maximum 1.6.x
```
why list the versions for everything else and not this?

---

This was initially only for node_exporter. Forgot to update the original line. Thanks